po/it.po

```diff
@@ -1252,49 +1211,43 @@ msgstr "Animazione apertura nuove finestre"
 
 #: ui/SettingsFineTune.ui.h:13
 msgid "Gnome functionality"
-msgstr ""
+msgstr "Funzionalità Gnome"
 
 #: ui/SettingsFineTune.ui.h:14
-#, fuzzy
 msgid "Keep original gnome-shell dash"
-msgstr "Mantieni dash originale di gnome-shell (schermata panoramica)"
+msgstr "Mantieni dash originale di gnome-shell"
 
 #: ui/SettingsFineTune.ui.h:15
 msgid "(overview)"
-msgstr ""
+msgstr "(panoramica)"
 
 #: ui/SettingsFineTune.ui.h:16
 msgid "Keep original gnome-shell top panel"
 msgstr "Mantieni il pannello superiore della gnome-shell originale"
 
 #: ui/SettingsFineTune.ui.h:17
-#, fuzzy
 msgid "Activate panel menu buttons on click only"
-msgstr ""
-"Attiva i pulsanti del menu del pannello (ad es. Menu della data) solo al clic"
+msgstr "Attiva i pulsanti del menu del pannello solo con un clic"
 
 #: ui/SettingsFineTune.ui.h:18
-#, fuzzy
 msgid "(e.g. date menu)"
-msgstr "Data e ora"
+msgstr (es. menu data)"
```

Comment:
"make" can't complete due to the missing quote symbol
I've made PR #2066 for that